charts/csi-cloudscale/templates/storageclass.yaml

```diff
@@ -0,0 +1,24 @@
+{{- $csi := .Values.csi }}
+{{- $provisioner := .Values.provisioner }}
+{{- range $storageClass := .Values.csi.storageClasses }}
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: {{ $storageClass.name }}
+{{- if $storageClass.default }}
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
+{{- end }}
```

Comment:
Adding labels would make sense here. In example `name: ssd`.

Reply:
Since this is not directly related to the Helm chart I created #46 for this.